docs/en/sql-reference/functions/random-functions.md

```diff
@@ -23,6 +23,68 @@ Uses a linear congruential generator.
 
 ## randConstant {#randconstant}
 
-Returns a pseudo-random UInt32 number, The value is one for different blocks.
+Produces a constant column with a random value.
+
+The function is intended for service purposes.
+
+**Syntax**
+
+``` sql
+randConstant(x)
+```
+
+**Parameters**
+
+-   `x` — optional expression. Its value is ignored and used only for [Common Subexpression Elimination](index.md#common-subexpression-elimination).
+
+**Returned value**
+
+-   Pseudo-random number.
+
+Type: [UInt32](../data-types/int-uint.md).
+
+**Examples**
```

Comment:
I think we need to emphisize difference with `rand`, and provide one comparative example, like ` SELECT rand(), rand(1), rand(number), randConstant(), randConstant(1), randConstant(number) FROM numbers(3)`. All other examples are covered by this one.